root/edit/details/EditPlace.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * @flow
+ * Copyright (C) 2019 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import DescriptiveLink from '../../static/scripts/common/components/DescriptiveLink';
+import Diff from '../../static/scripts/edit/components/edit/Diff';
+import FullChangeDiff from '../../static/scripts/edit/components/edit/FullChangeDiff';
+import FullChangeDiffContent from '../../static/scripts/edit/components/edit/FullChangeDiffContent';
+import {formatCoordinates} from '../../utility/coordinates';
+import formatDate from '../../static/scripts/common/utility/formatDate';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type PlaceEditT = {
+  ...EditT,
+  display_data: {
+    address: CompT<string>,
+    area: CompT<AreaT>,
+    begin_date: CompT<PartialDateT>,
+    comment: CompT<string>,
+    coordinates: CompT<CoordinatesT>,
+    end_date: CompT<PartialDateT>,
+    ended: CompT<boolean>,
+    name: CompT<string>,
+    place: PlaceT,
+    type: CompT<PlaceTypeT>,
+  },
+};
+
+const EditPlace = ({edit}: {edit: PlaceEditT}) => {
+  const display = edit.display_data;
+  const place = display.place;
+  let notDiffContent = true;
+  return (
+    <table className="details edit-place">
+      <tbody>
+        <tr>
+          <th>{addColonText(l('Place'))}</th>
+          <td><DescriptiveLink entity={place} /></td>
+        </tr>
+
+        {display.name ? (
+          <Diff
+            label={addColonText(l('Name'))}
+            newText={display.name.new}
+            oldText={display.name.old}
+            split="\s+"
+          />
+        ) : null}
+        {display.comment ? (
+          <Diff
+            label={addColonText(l('Disambiguation'))}
+            newText={display.comment.new}
+            oldText={display.comment.old}
+            split="\s+"
+          />
+        ) : null}
+        {display.type ? (
+          <FullChangeDiff
+            label={addColonText(l('Type'))}
+            newText={display.type.new.name}
+            oldText={display.type.old.name}
```

Comment:
I'm expecting these two should be localized? @mwiencek, please confirm.

Reply:
Well I've created consts oldType and newType just incase. Hope that resolved this :)

Reply:
By localized he meant translated, using `lp_attributes`. I'm not sure the current TT template does that, actually, but we should now in any case. Suggested that again in https://github.com/metabrainz/musicbrainz-server/pull/1229#discussion_r355173889

Reply:
ohh